.gitlab/bazel/defs.yaml

```diff
@@ -30,6 +30,7 @@
   retry: 0
   variables:
     GIT_DEPTH: 2
+    BAZEL_OUTPUT_USER_ROOT: "C:/tmpbzl"
```

Comment:
May I kindly ask for DataDog/datadog-agent-buildimages#992 to go through prior to deciding whether to merge?

Reply:
Friendly ping on some questions.
1. Does this make sense with the max_path fix?  That is for our containers, so we'll want developer setup instructions to enable the same on builds on raw windows.
2. Does this even make sense?